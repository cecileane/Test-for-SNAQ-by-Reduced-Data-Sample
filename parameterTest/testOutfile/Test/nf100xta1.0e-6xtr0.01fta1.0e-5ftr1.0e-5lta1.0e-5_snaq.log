optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=1.0e-5,
                       xtolAbs=1.0e-6, xtolRel=0.01.
 max number of failed proposals = 100, liktolAbs = 1.0e-5.
rootname for files: nf100xta1.0e-6xtr0.01fta1.0e-5ftr1.0e-5lta1.0e-5_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 41563
seed: 41563 for run 1
Sat Oct  1 17:07:07 2016

 BEGIN SNaQ for run 1, seed 41563 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED for not having more moves to propose: movesfail [0,25,25,4,0,61], Nmov [217,25,25,4,10000,61]
END optTopLevel: found minimizer topology at step 197 (failures: 91) with -loglik=127.5115 and ht_min=[0.45392,10.0,0.0,0.5186,2.23709,0.0,0.36548]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 4		 4		 1	 |	 NA 		 NA 		 NA 
mvorigin	 41		 41		 3	 |	 0		 0		 --
mvtarget	 51		 51		 3	 |	 0		 0		 --
chdir		 16		 16		 0	 |	 1		 1		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 84		 53		 1	 |	 1		 1		 --
Total		 196		 165		 8	 |	 2		 2		 2
Proportion	 -- 		 1.0		 0.0	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 1, -loglik of best 127.51150018543095
((6,(1,(2,((3,4):0.0)#H7:0.0::0.5460844792623192):0.5185996314773672):2.237093348517682):10.0,5,#H7:0.36548052376198936::0.4539155207376809);
---------------------
seed: 13615 for run 2
Sat Oct  1 17:07:21 2016

 BEGIN SNaQ for run 2, seed 13615 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED for not having more moves to propose: movesfail [0,25,25,4,0,61], Nmov [217,25,25,4,10000,61]
END optTopLevel: found minimizer topology at step 229 (failures: 82) with -loglik=127.5115 and ht_min=[0.45392,10.0,0.0,0.5186,2.23709,0.36548,0.0]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 2		 2		 1	 |	 NA 		 NA 		 NA 
mvorigin	 54		 54		 5	 |	 0		 0		 --
mvtarget	 60		 60		 2	 |	 0		 0		 --
chdir		 19		 19		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 93		 41		 0	 |	 5		 5		 --
Total		 228		 176		 9	 |	 5		 5		 5
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 2, -loglik of best 127.51150018563334
((6,(1,(2,((3,4):0.0)#H7:0.0::0.5460839918467895):0.5186025059182431):2.237094458769108):10.0,5,#H7:0.3654771111517835::0.4539160081532106);
---------------------

Sat Oct  1 17:07:37 2016
MaxNet is ((6,(1,(2,((3,4):0.0)#H7:0.0::0.5460844792623192):0.5185996314773672):2.237093348517682):10.0,5,#H7:0.36548052376198936::0.4539155207376809); 
with -loglik 127.51150018543095
