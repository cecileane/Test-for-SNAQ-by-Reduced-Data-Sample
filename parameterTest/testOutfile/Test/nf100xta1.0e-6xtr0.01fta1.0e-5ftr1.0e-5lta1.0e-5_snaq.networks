((6,(1,(2,((3,4):0.0)#H7:0.0::0.5460844792623192):0.5185996314773672):2.237093348517682):10.0,5,#H7:0.36548052376198936::0.4539155207376809);, with -loglik 127.51150018543095 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(3,4,(((6)#H7:::0.6823266294774417,5):2.163185412740178,((1,#H7:::0.3176733705225583):2.349831950644277,2):0.8394802517309187):0.0);, with -loglik 177.05190118360557
(3,4,(((6,#H7:::0.2609103744103214):9.127591800230508,5):0.7028613860687408,((1)#H7:::0.7390896255896786,2):1.5208141368891486):0.0);, with -loglik 179.04874906861335
(3,4,(((6,(1,(2)#H7:::0.523485984088964):3.3993209223675143):0.10541762318293567,5):0.18875154078624654,#H7:::0.47651401591103604):0.0);, with -loglik 286.18220863134735
(3,4,((5)#H7:::0.5711276529653314,((1,(6,#H7:::0.4288723470346686):6.609218887284536):0.018156422429543035,2):0.36188267460412804):0.0);, with -loglik 299.59948707568907
