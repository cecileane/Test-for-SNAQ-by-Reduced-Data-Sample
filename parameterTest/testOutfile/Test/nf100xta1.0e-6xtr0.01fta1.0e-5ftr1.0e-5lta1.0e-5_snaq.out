((6,(1,(2,((3,4):0.0)#H7:0.0::0.5460844792623192):0.5185996314773672):2.237093348517682):10.0,5,#H7:0.36548052376198936::0.4539155207376809); -Ploglik = 127.51150018543095
 Dendroscope: ((6,(1,(2,((3,4):0.0)#H7:0.0):0.5185996314773672):2.237093348517682):10.0,5,#H7:0.36548052376198936);
 Elapsed time: 29.98386102 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 ((6,(1,(2,((3,4):0.0)#H7:0.0::0.5460844792623192):0.5185996314773672):2.237093348517682):10.0,5,#H7:0.36548052376198936::0.4539155207376809);, with -loglik 127.51150018543095
 ((6,(1,(2,((3,4):0.0)#H7:0.0::0.5460839918467895):0.5186025059182431):2.237094458769108):10.0,5,#H7:0.3654771111517835::0.4539160081532106);, with -loglik 127.51150018563334
-------
